docs/identities/model/identity-schema-selection.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: identity-schema-selection
+title: Identity Schema Selection
+---
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+```
+
+# Identity schema selection for self-service registration and login flows
+
+Ory Kratos now supports selecting an identity schema during **registration** and **login** self-service flows by specifying a
+query parameter. This gives administrators more control over which schemas are used and allows multi-tenant or multi-profile
+setups with distinct identity models.
+
+This feature is available in **Ory Kratos OEL** for now and will be available in Ory Network soon.
```

Comment:
```suggestion

```

I am not sure if we should have this here, because we will 100% forget to update it later on. 

Instead can we use a reusable text snippet like [this one](https://github.com/ory/docs/blob/master/docs/_common/need-help.mdx), that says something like "this feature is in preview, please reach out if you are interested" or something like that - for features that are not released in all versions of Ory yet. 
IDK how you feel about this @unatasha8, there is probably a number of ways we can solve this.